vpr/src/route/partition_tree.h

```diff
@@ -25,6 +25,19 @@ inline Side operator!(const Side& rhs) {
     return Side(!size_t(rhs));
 }
 
+/** Part of a net. Used by \ref DecompNetlistRouter. */
```

Comment:
Add file scope doxygen comment saying what a partition tree is and what it is for.

Reply:
Define a virtual net more precisely.  Defined by the original net and a bounding box that contains the sinks to route and any partial routing from which new connection can branch off?